### PR TITLE
Added mixed_enzymes property

### PR DESCRIPTION
**Pull request by andreacosolo**
In case an Enzyme item is a mix of multiple enzymes, this property is used to link to the single Enzymes that belong to the mix, where all the related metadata is captured.

### CODE REVIEW

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808&bid=18557438)

File: src/encoded/schemas/enzyme.json [hunk 1]
```diff
@@ -63,18 +63,31 @@
             "pattern": "^[ATGCYRWSKMDVHBN]+$"
         },
         "site_length": {
-          "title": "Site Length",
-          "description": "The length of the enzyme recognition sequence.",
-          "type": "integer",
-          "lookup": 70,
-          "internal_comment": "this can be calculated if a recognition sequence is provided but can be set even without recognition sequence"
+            "title": "Site Length",
```
Comment (j1z0) [inline src/encoded/schemas/enzyme.json:66]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808&bid=18557438)

File: src/encoded/schemas/enzyme.json (hunk 1 above)
Comment (j1z0) [inline src/encoded/schemas/enzyme.json:66]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808)

File: src/encoded/schemas/enzyme.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "Enzyme",
-    "description": "DNA digestion enzymes like HindIII or DNaseI",
+    "description": "DNA restriction enzymes like HindIII or DNaseI",
```
Comment (j1z0) [inline src/encoded/schemas/enzyme.json:3]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808&bid=18557438)

File: src/encoded/schemas/enzyme.json [hunk 3]
```diff
@@ -63,18 +63,32 @@
             "pattern": "^[ATGCYRWSKMDVHBN]+$"
         },
         "site_length": {
-          "title": "Site Length",
-          "description": "The length of the enzyme recognition sequence.",
-          "type": "integer",
-          "lookup": 70,
-          "internal_comment": "this can be calculated if a recognition sequence is provided but can be set even without recognition sequence"
+            "title": "Site Length",
```
Comment (j1z0) [inline src/encoded/schemas/enzyme.json:66]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808)

File: src/encoded/schemas/enzyme.json (hunk 2 above)
Comment (j1z0) [inline src/encoded/schemas/enzyme.json:3]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808)

Comment (sbreiff) [inline src/encoded/schemas/enzyme.json:3]: Why the change from digestion to restriction? Micrococcal nuclease isn't a restriction enzyme, for example.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808&bid=18557438)

File: src/encoded/schemas/enzyme.json (hunk 3 above)
Comment (j1z0) [inline src/encoded/schemas/enzyme.json:66]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808)

File: src/encoded/schemas/enzyme.json (hunk 2 above)
Comment (andreacosolo) [reply src/encoded/schemas/enzyme.json]: Oh, now I see why it was there in the first place. Then endonuclease is what we mean. Digestion enzyme didn't sound right, but I understand now. I reverted that commit.

File: src/encoded/schemas/enzyme.json [hunk 4]
```diff
@@ -63,18 +63,32 @@
...
+        "mixed_enzymes": {
+            "title": "Mixed Enzymes",
+            "description": "The enzymes that are combined",
+            "comment": "If this is a multi-enzyme item, list here each single Enzyme item",
+            "type": "array",
+            "lookup": 90,
+            "uniqueItems": true,
+            "items": {
+                "title": "Enzyme",
+                "description": "The restriction enzyme",
```
Comment (sbreiff) [inline src/encoded/schemas/enzyme.json:88]: Could consider changing this to "digestion" as well, though most enzyme combinations will probably be restriction enzymes so I don't think it matters that much.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808&bid=18557438)

File: src/encoded/schemas/enzyme.json (hunk 3 above)
Comment (j1z0) [inline src/encoded/schemas/enzyme.json:66]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5668808)
